frontend/cypress.json

```diff
@@ -1,3 +1,3 @@
 {
-    "baseUrl": "http://localhost:8081"
+    "baseUrl": "http://localhost:8080"
```

Comment:
This change may affect the capability of running the tests for some people if they have something running on that port already. Is there a fix for this?

Reply:
Yes, you are right. If you are not running on port 8080, cypress can not verify that the server is running. This gets fixed once docker gets set up

Reply:
Ok, we'll make sure to have docker set up in the next sprint so that we can fix issues like this.